File: brain/53af253a-0c1c-4635-9416-1311c06562ef/scratch/replace_colors.py
import os

def replace_emerald_with_green(directory):
    for root, dirs, files in os.walk(directory):
        for file in files:
            if file.endswith('.vue'):
                file_path = os.path.join(root, file)
                with open(file_path, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                new_content = content.replace('emerald', 'green')
                
                if new_content != content:
                    with open(file_path, 'w', encoding='utf-8') as f:
                        f.write(new_content)
                    print(f"Updated: {file_path}")

if __name__ == "__main__":
    replace_emerald_with_green(r'c:\XAMPP\htdocs\EODS\Master\frontend\src')

File: brain/53af253a-0c1c-4635-9416-1311c06562ef/scratch/replace_colors.js
const fs = require('fs');
const path = require('path');

const colorsToReplace = ['emerald', 'teal', 'indigo', 'purple', 'orange', 'amber'];

function walk(dir, callback) {
  fs.readdirSync(dir).forEach( f => {
    let dirPath = path.join(dir, f);
    let isDirectory = fs.statSync(dirPath).isDirectory();
    isDirectory ? walk(dirPath, callback) : callback(path.join(dir, f));
  });
};

walk('c:/XAMPP/htdocs/EODS/Master/frontend/src', (filePath) => {
  if (filePath.endsWith('.vue')) {
    let content = fs.readFileSync(filePath, 'utf8');
    let newContent = content;
    
    colorsToReplace.forEach(color => {
      const regex = new RegExp(color, 'g');
      newContent = newContent.replace(regex, 'green');
    });

    if (content !== newContent) {
      fs.writeFileSync(filePath, newContent, 'utf8');
      console.log('Updated:', filePath);
    }
  }
});
